acceptance/git/git.go

```diff
@@ -240,6 +241,12 @@ func createGitRepository(ctx context.Context, repositoryName string, files *godo
 			return err
 		}
 
+		// Replace ${GITHOST} with the real git host.
+		// Used in acceptance/examples/default-policy.yaml
+		// Hopefully a noop for all other files.
+		re := regexp.MustCompile(regexp.QuoteMeta("${GITHOST}"))
+		b = []byte(re.ReplaceAllString(string(b), Host(ctx)))
```

Comment:
Add a test for this, I'm fairly certain that `Host()` will return `""` at this point, i.e. before [line 177](https://github.com/simonbaird/ec-cli/blob/48fb0aadf463ab2ed95697e0db4528a0ea44f336/acceptance/git/git.go#L177)

Reply:
From what I can tell, it's fine since `startStubGitServer` via "Given stub git daemon running" calls that code, and that happens early on.